itou/www/approvals_views/forms.py

```diff
@@ -202,6 +162,26 @@ def __init__(self, *args, approval, siae, **kwargs):
 
         # Customize "reason" field
         self.fields["reason"].choices = ProlongationReason.for_company(self.instance.declared_by_siae)
+        other_prolongation_duration = Prolongation.objects.get_cumulative_duration_for(
+            approval.pk, ignore=self.instance.pk
+        )
```

Comment:
La mutation suivante ne fait casser aucun test.
```suggestion
        other_prolongation_duration = Prolongation.objects.get_cumulative_duration_for(approval.pk)
```

Reply:
Effectivement car on est toujours en mode création et `self.instance.pk` est toujours `None`.
Je suis tenté de le laisser car il ne coûte rien et cela facilitera l'éventuelle réutilisation de ce formulaire pour l'édition ?

Reply:
Je serais tenté de le supprimé, ou au moins de le commenter, autrement on se demande pourquoi il est là.

Reply:
Tu as raison je l'ai supprimé, il y a peu de chance qu'on réutilise le formulaire tel quel.